test/langtools/tools/javac/patterns/CastConversionMatch.java

```diff
@@ -3,7 +3,7 @@
  * @bug 8231827
  * @summary Match which involves a cast conversion
  * @compile/fail/ref=CastConversionMatch.out -XDrawDiagnostics CastConversionMatch.java
- */
+ * @compile --enable-preview --source ${jdk.version} CastConversionMatch.java */
```

Comment:
@enablePreview?

Reply:
I have two compile commands. Wanted to test one with and one without preview.

Reply:
right missed that sorry